src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -795,7 +796,11 @@ public boolean isElementCovered(WebElement element) {
     }
 
     public void verifyUnclickable(WebElement element) {
-        assertNotNull(element.getAttribute("disabled"));
+        try {
+            assertTrue(element.getAttribute("class").contains("disabled"));
+        } catch (AssertionError e) {
+            assertNotNull(element.getAttribute("disabled"));
+        }
```

Comment:
This logic would be better: if element is an anchor, check for class, else check for attribute.